articles/extensions/authorization-extension/v2/import-export-data.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Importing Data Into and Exporting Data from the Authorization Extension
+description: How to import/export Authorization Extension Data
+---
+
+# Authorization Extension: Import/Export Data
+
+You can import new data from or export existing authorization data to a JSON file. This can be useful when moving environments. However, remember that roles and permissions are linked to specific clients, so you'll need to update the correct `clientId`.
```

Comment:
The `However,` sentence is repeated also at the warning at the end. Maybe remove this sentence and place the warning here?